cfep-23.md

```diff
@@ -0,0 +1,73 @@
+
+The integration in the Element ecosystem was seen as exciting and positive: emoji reactions were available for the first time, and threads were shortly implemented too. However, over time the excitement faded away, which prompted the proposal discussed in this CFEP.
+
+## Motivation
+
+Element/Matrix offers a free to use real-time chat platform with web, desktop and mobile clients with end-to-end encryption messaging. Each community or space can create one or more rooms, public or private, to discuss different topics. Other features like videocalls, private messaging, and file sharing are available. Users can create a new account or log in with their Github credentials, among others.
+
+However, it suffers from a few issues that limit its usability within our community. The main concerns are:
+
+- End-to-end encryption fails sometimes, leaving the users unable to read previous messages, or have their messaged read by others. Synchronization of keys across clients is necessary to fix it, but in some other cases the issue has dissapeared spontaneously.
+- Some messages are considered unread for months at a time, and there's no way to find them and mark them as read.
+- The thread system offers very limited ways of tracking conversations and making them discoverable. It's very difficult to follow on topics started hours ago, let alone days or weeks ago.
+- Scoped discussions require a lot of discipline from the users, since threads are optional, and the real-time chat is preferred in the UI.
+
+Zulip, on the other side, is a free to use, open source, real-time discussion board. It has web, desktop and mobile clients. Each community has its Zulip instance, where several "channels" can be created. Each channel hosts a number of threads or topics. Instant messaging is also available within the Zulip instance. Users can also create a new account or log in with their Github credentials, among others.
+
+Zulip offers some features we would consider beneficial to the conda-forge community:
+
+- Threads (or topics) are encouraged and the default user experience.
+- Each topic must have a title that promotes well-scoped discussions, which can be marked as solved when the issue has been addressed.
+- Moderation tools are available to split threads into new ones in case the conversation goes off-topic.
+- Users can "follow" threads they find interesting, which has its own dedicated view. This makes it easier to track updates to conversations you have started or participated in.
+- Private rooms are also available, but they don't offer end-to-end encryption. For this matter, a sensitive communications protocol is proposed below.
+
+## Proposal
+
+After accepting this proposal, we will implement the following decisions:
+
+- The Zulip instance published at `conda-forge.zulipchat.com` will be open to the public with world-readable permissions.
```

Comment:
@jaimergp, you mention the Zulip instance across this doc, but there is no clear definition. My main question when reading the above is whether this `Zulip instance` will work as elementary does right now or if it will be a self-hosted chat service (if it does, where will it reside)?

Reply:
Zulipchat.com subdomains are hosted by Zulip, so no, we are not self hosting.